docs/source/lab/simulation.rst

```diff
@@ -11,3 +11,12 @@ See the following resources for  information on running a simulation:
 - :ref:`ASReview command line interface for simulation <API/cli:Simulate>`
 - :doc:`../guides/simulation_study_results`
 - :doc:`../guides/sim_overview`
+
+
+.. warning::
+
+    If you upload your own data, make sure to remove duplicates and to retrieve 
+    as many abstracts as possible (`don't know how?
+    <https://asreview.nl/the-importance-of-abstracts/>`_). With clean data you
+    benefit most from what :doc:`active learning <../guides/activelearning>` 
+    has to offer you. 
```

Comment:
Maybe this is not the right place, as the ideal situation, the cleaning has happened before you run a simulation. Then, a simulation shows your process as is. On the other hand, I see the value of mentioning the GIGO principle here.